tasks/src/main/scala/sbt/ExecuteProgress.scala

```diff
@@ -14,28 +14,32 @@ import sbt.internal.util.RMap
  * except `started` and `finished`, which is called from the executing task's thread. All methods
  * should return quickly to avoid task execution overhead.
  */
-trait ExecuteProgress[F[_]] {
+trait ExecuteProgress {
```

Comment:
I think this is an overall good thing for sbt 2.x, but I want to loop in Gradle folks /cc @dragos, @Duhemm.